src/test/service/contentServiceSpec.js

```diff
@@ -113,6 +113,12 @@ describe('Composite search services', function () {
     })
     done()
   })
+
+  it('check for metafilter in config with whitelist', function (done) {
```

Comment:
not sure what this test is testing. could you add some details?

Reply:
Details added